platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/maplayout/DebugModeActivity.java

```diff
@@ -47,9 +49,21 @@ protected void onCreate(Bundle savedInstanceState) {
       @Override
       public void onMapReady(@NonNull MapboxMap map) {
         mapboxMap = map;
+
+        mapboxMap.getUiSettings().setZoomControlsEnabled(true);
+
+        // show current zoom level on screen
+        final TextView textView = (TextView) findViewById(R.id.textZoom);
+        mapboxMap.setOnCameraChangeListener(new MapboxMap.OnCameraChangeListener() {
+          @Override
+          public void onCameraChange(CameraPosition position) {
+            textView.setText(String.format("Zoom : %s", position.zoom));
```

Comment:
when it comes to readability, it think showing where %s is located for String.format is much clearer else you need to jump to the constant field and check what the value is. Putting it in the constant is making readability worse. The correct solution would be to put this in strings.xml as android studio shows you the value directly in the IDE.